Solutions/Marain.UserNotifications.ApiDeliveryChannel.Host/Marain.UserNotifications.ApiDeliveryChannel.Host.csproj

```diff
@@ -32,15 +32,8 @@
       <CopyToPublishDirectory>Never</CopyToPublishDirectory>
     </None>
     <None Update="local.settings.template.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
```

Comment:
Don't block releasing this, but should we copy the template to the publish directory?

Reply:
Argh. I already fixed one of these but missed this. Added https://github.com/marain-dotnet/Marain.UserNotifications/issues/271